clap-v3-utils/src/input_parsers/signer.rs

```diff
@@ -15,6 +18,90 @@ use {
 // Sentinel value used to indicate to write to screen instead of file
 pub const STDOUT_OUTFILE_TOKEN: &str = "-";
 
+#[derive(Debug)]
+pub struct SignerSourceParserBuilder {
+    prompt: bool,
+    file_path: bool,
+    usb: bool,
+    stdin: bool,
+    pubkey: bool,
```

Comment:
Could we add the `allow_` prefix to these? I keep confusing myself about what these mean.

Reply:
Yes, good idea!